src/map/mail.cpp

```diff
@@ -418,3 +419,39 @@ void mail_refresh_remaining_amount( struct map_session_data* sd ){
 		sc_start2( &sd->bl, &sd->bl, SC_DAILYSENDMAILCNT, 100, doy, 0, -1 );
 	}
 }
+
+void npc_mail_send(struct npc_data *nd, const char *dest_name, const char *title, const char *body_msg, int body_len, int zeny) {
+	struct mail_message msg;
+
+	if (body_len > MAIL_BODY_LENGTH)
+		body_len = MAIL_BODY_LENGTH;
+
+	msg.zeny = zeny;
+	//msg.item[1].nameid = 502;
+	//msg.item[1].amount = 1;
+	for (int i = 0; i < MAIL_MAX_ITEM; i++) {
+		msg.item[i].nameid = 0;
+		msg.item[i].amount = 0;
+	}
+	msg.id = 0; // id will be assigned by charserver
+	msg.send_id = nd->bl.id;
+	msg.dest_id = 0; // will attempt to resolve name
+	safestrncpy(msg.send_name, nd->name, NAME_LENGTH);
+	safestrncpy(msg.dest_name, (char*)dest_name, NAME_LENGTH);
+	safestrncpy(msg.title, (char*)title, MAIL_TITLE_LENGTH);
+	msg.type = MAIL_INBOX_NORMAL;
```

Comment:
how about adding this to parameters list?
The whole function will be agiler if will be added the type options to the parameters...
So people can decide what type of mail they wish to send 🍅